### DISCUSSION TITLE
po5 GitHub account deleted?

### DISCUSSION THREAD

**Discussion by xfzv** [tomasklaen/uosc#619] [General]
Not directly related to uosc but considering the README mentions `thumbfast`, I thought I would ask here:

https://github.com/tomasklaen/uosc/blob/d25f3e2e9962fe58939515357931229faa6719e4/README.md?plain=1#L16

https://github.com/tomasklaen/uosc/blob/d25f3e2e9962fe58939515357931229faa6719e4/README.md?plain=1#L81

I noticed today that https://github.com/po5/ no longer exists.

Does anyone have more info?

**Comment by hrxn** [comment]
Not sure, I also see only the 404 page, let's hope this is a temporary issue?

**Comment by tomasklaen** [comment]
No idea. This is definitely disconcerting 👀.

**Comment by christoph-heinrich** [comment]
I have been wondering why they haven't responded to [my PR to memo](https://github.com/christoph-heinrich/memo/tree/tvshow_menu) for 3 weeks and why there was no progress on the evafast rewrite after I fixed that floating point bug, but I saw them respond to some mpv issue a few days ago, so it's not like they were completely inactive.

I guess the best way forward in case this isn't temporary is to figure out who has the most up to date fork so that anyone who wants to can develop from there?

[thumbfast](https://github.com/christoph-heinrich/thumbfast) is probably up to date.
The [evafast rewrite](https://github.com/christoph-heinrich/evafast/commits/rewrite_fix_stuck_speed) I have up to the floating point fix, and from what I remember the commits after that didn't really do anything functionality wise.
I'm pretty sure [memo](https://github.com/christoph-heinrich/memo) is up to date.

**Reply by xfzv** [reply]
Thanks for the heads up about your forks. Other possible alternatives:

Original plugin | Alternative
-- | --
po5/chapterskip | [Eisa01/SmartSkip.lua](https://github.com/Eisa01/mpv-scripts/blob/master/scripts/SmartSkip.lua)
po5/mpv_sponsorblock | [TheCactusVert/mpv-sponsorblock](https://github.com/TheCactusVert/mpv-sponsorblock) or [jouni/mpv_sponsorblock_minimal](https://codeberg.org/jouni/mpv_sponsorblock_minimal)

I haven't tried any of these yet but they should do the job I guess.

**Reply by christoph-heinrich** [reply]
We should replace the links in [User Scripts](https://github.com/mpv-player/mpv/wiki/User-Scripts) once we know the best forks so users have some way of getting those scripts.
There was already an [mpv issue](https://github.com/mpv-player/mpv/issues/12350) about that.

**Reply by haasn** [reply]
For chapterskip there is also https://github.com/haasn/gentoo-conf/blob/xor/home/nand/.mpv/scripts/avail/skipchapters.lua

It seems more similar in scope

**Reply by christoph-heinrich** [reply]
@llyyr pointed out in IRC that https://github.com/gaesa/mpv-thumbfast seems to be up to date + 2 commits.
I've updated master on my fork to that and removed the additional commits to get back to po5's state.

**Reply by xfzv** [reply]
@haasn Thank you, [skipchapters.lua](https://github.com/haasn/gentoo-conf/blob/xor/home/nand/.mpv/scripts/avail/skipchapters.lua) is a very similar alternative indeed.

@christoph-heinrich just a heads-up/question regarding `evafast`: I'm using the `rewrite` branch and I noticed I'm getting the following warning when using `lookahead` option in `evafast.conf`:

```console
mpv --no-really-quiet /path/to/video
[evafast] script-opts/evafast.conf:31 unknown key 'lookahead', ignoring
```

Using `subs_lookahead` instead fixes the warning. I guess the `rewrite` branch from the original repository didn't have the updated option name in `evafast.conf`?

- https://github.com/christoph-heinrich/evafast/blob/rewrite/evafast.lua#L45-L46
- https://github.com/christoph-heinrich/evafast/blob/rewrite/evafast.conf#L30-L31

**Reply by christoph-heinrich** [reply]
@xfzv The conf wasn't updated [when that was changed](https://github.com/christoph-heinrich/evafast/commit/29fb0e4b08df649a04f112f8d213920028d7128f) in the lua file.

**Comment by iTzHard** [comment]
Hello there, official word from po5 here:

"I got shadowbanned and I'm trying to find out a solution, I haven't disappeared. This should *hopefully* be temporary. I might move my stuff over to https://github.com/l-jared, will see what pans out."

I was the one to point out that the repo was suspended to po5, and we're currently investigating a solution.

**Reply by xfzv** [reply]
Good news! thanks for sharing.

**Reply by l-jared** [reply]
Stuff is back up, I do have issues and PRs saved locally but that'll do for now.  
Hope we get the po5 account back :^)

Feel free to clone repos in case these disappear too.

**Reply by xfzv** [reply]
Welcome back!

Just to make sure: if the po5 account is restored, should we keep using it as if nothing happened or do you intend to switch to this one (l-jared) regardless?

**Reply by l-jared** [reply]
This is just a temporary solution, I do want to go back to po5, or transfer the repos to an organization so this doesn't happen again (and stuff will keep working thanks to redirects when transferring ownership). Sadly they've locked me out of the transfer feature as well.

**Reply by tomasklaen** [reply]
Just curious. What's the shadow ban for? What happened ? :)

**Reply by po5** [reply]
Bullying and harrassement. We are back (:

![image](https://github.com/tomasklaen/uosc/assets/42466980/06910cac-7e3f-4d4d-9c4b-1f937c9efcb3)

**Reply by hrxn** [reply]
good news!
